pkg/services/rpcsrv/params/param.go

```diff
@@ -443,3 +443,12 @@ func (p *Param) GetUUID() (uuid.UUID, error) {
 	}
 	return id, nil
 }
+
+// GetRawMessage returns underlying JSON raw parameter. Do not modify the
+// resulting JSON, otherwise it may affect the parameter decoding.
+func (p *Param) GetRawMessage() (json.RawMessage, error) {
+	if p == nil {
+		return nil, errMissingParameter
+	}
+	return p.RawMessage, nil
```

Comment:
This will make CodeQL unhappy because it's a direct user input used for logging.

Reply:
For now, just drop it. Some parameters may also be pretty big, not something to be logged entirely.